packages/sdk-react/src/core/queries/useOnboardingDocuments.ts

```diff
@@ -1,77 +1,22 @@
-import type { ErrorType } from '@/core/queries/types';
-import {
-  AllowedCountries,
-  ApiError,
-  type EntityOnboardingDocumentsPayload,
-  type OnboardingDocumentsDescriptions,
-  type PersonOnboardingDocumentsPayload,
-} from '@monite/sdk-api';
-import { useMutation, useQuery } from '@tanstack/react-query';
+import { components } from '@/api';
 
 import { useMoniteContext } from '../context/MoniteContext';
 
-export const ONBOARDING_DOCUMENT_QUERY = 'onboarding_documents';
+export const useDocumentDescriptions = (
+  country: AllowedCountries | undefined
+) => {
+  const { api } = useMoniteContext();
 
-const documentQueryKeys = {
-  all: () => [ONBOARDING_DOCUMENT_QUERY],
-  descriptions: (country?: AllowedCountries) =>
-    country
-      ? [...documentQueryKeys.all(), 'descriptions', country]
-      : [...documentQueryKeys.all(), 'descriptions'],
-};
-
-export const useCreateEntityDocuments = () => {
-  const { monite } = useMoniteContext();
-
-  return useMutation<
-    EntityOnboardingDocumentsPayload,
-    ApiError,
-    EntityOnboardingDocumentsPayload
-  >({
-    mutationFn: async (payload) => {
-      await monite.api.onboardingDocuments.createEntityDocuments(payload);
-
-      return payload;
+  return api.frontend.getFrontendDocumentTypeDescriptions.useQuery(
+    {
+      query: {
+        country: country!,
```

Comment:
we discussed that `!` is a bad practice, but we can't fall back on an empty string here. Should we use for example 
 'US' or `!` is legal for such cases?

Reply:
This seems to be exactly the one case that is the exception.
If something goes wrong, I would prefer _an empty value to be sent_ rather than a fallback in the form of `US`.